README.md

```diff
@@ -110,7 +105,7 @@ by default the themes path goes here **/openedx/themes**
 ## Default themes
 These themes will be installed in a default installation.
 
-- bragi
+- [bragi](https://github.com/eduNEXT/ednx-saas-themes/tree/edunext/limonero.master)
```

Comment:
Maybe this reference isn't necessary due to edx-saas-themes is a private repo

Reply:
@dcoa I think that ednx-saas-themes will be a public repository. @felipemontoya Am I wrong?

Reply:
Mafer recently opened this as a discussion internally. I do think that this repo will become public, but we need to let that discussion take its course. It could be that it is very fast.

Reply:
It will be public, so I will close this conversation and merge.